site/includes/metadata.hbs

```diff
@@ -1 +1,18 @@
-<meta name="description" content="{{#if page.description }}{{page.description}}{{else}}{{{i18n "site-description"}}}{{/if}}" />
+<meta name="description" content="{{#if page.description}}{{page.description}}{{else}}{{{i18n "site-description"}}}{{/if}}" />
+
+{{#if canonical}}<link rel="canonical" href="{{canonical}}" />{{/if}}
+
+{{#if altLangPrefix}}
+
+{{#is language "en"}}
+<link rel="alternate" hreflang="en" href="{{pagename}}" />
```

Comment:
this seems redundent information compared to the canonical if the canonical is english